apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/transaction/TransactionJsonSerializer.java

```diff
@@ -15,5 +15,15 @@
 
     JSONObject toJson(Transaction transaction);
 
+    /**
+     * Only use for tests.
+     *
+     * @param transaction the given transaction
+     * @return serialized transaction as a JSONObject
+     * @deprecated
+     */
+    @Deprecated(forRemoval = true)
+    JSONObject toLegacyJsonFormat(Transaction transaction);
```

Comment:
To use only for tests, i would suggest just to copy-paste into a test-wide LegacyTransactionJsonSerializer desired method and that's it

Reply:
moved